ams/server/src/main/java/com/netease/arctic/server/ArcticManagementConf.java

```diff
@@ -313,6 +313,12 @@ public class ArcticManagementConf {
           .defaultValue(30)
           .withDescription("session timeout in minute");
 
+  public static final ConfigOption<Integer> TERMINAL_LOCAL_CORES =
```

Comment:
It seems to be a configuration only for the local terminal.
If yes, we may want to move it into `LocalSessionFactory` and name it `cores` because the `TerminalManager` will reorganize configurations based on different backend types.

You can find similar configurations only for the Kyuubi terminal in `KyuubiTerminalSessionFactory`.

Reply:
I agree that `LocalSessionFactory` is a better place for the config option. But do you think `cores` should be prefixed with the terminal type `local`, as it's only effective for the local terminal. That makes the full key `terminal.local.cores`.

Reply:
Sorry, my previous description was not clear. The complete configuration name in the configuration file should indeed be `terminal.local.cores`. Just as you have implemented, in the code, we only need to name the configuration `cores`, and `TerminalManager` will handle the decomposition of different backend configurations.